README.md

```diff
@@ -64,16 +64,24 @@ Now run one of the following commands:
 
 ### Previewing changes locally with Docker
 
-To preview local changes you've made to `teleport/docs` with Docker, you can run
+To preview local changes you've made to `teleport/docs` with Docker, try this script:
 
 ```bash
-NEXT_PATH=/abs/path/to/next
-TELEPORT_PATH=/abs/path/to/teleport
-SEM_VER=9.0 # Change this to whatever the latest version is
+#!/bin/bash
+DOCKER_IMAGE=node:16-slim
```

Comment:
The linter currently uses Node v14 in CI: https://github.com/gravitational/docs/blob/5c89232523e366fc79af174028d970a4ac6e53e9/.github/workflows/base.yml#L12

We have an outstanding issue to fix that: https://github.com/gravitational/docs/issues/181